Cargo.toml

```diff
@@ -96,7 +96,7 @@ prost-build = "0.13.3"
 # above this section.
 # --- dev-dependencies ---
 kitsune2_bootstrap_srv = { path = "crates/bootstrap_srv" }
-kitsune2_core = { path = "crates/core" }
+kitsune2_core = { version = "0.0.1-alpha", path = "crates/core" }
```

Comment:
Publish complains if you don't set a version for dependencies, fair enough!

Reply:
Can you use the workspace version here?

Reply:
I can't find anything in the docs about this and the `workspace.version` syntax is rejected by Cargo

Reply:
Okay